frontend/test/metabase/lib/expressions/__support__/shared.js

```diff
@@ -101,20 +101,30 @@ const aggregation = [
     ["avg", ["coalesce", total, tax]],
     "coalesce inside an aggregation",
   ],
+  [
+    "CountIf(49 <= [Total])",
+    ["count-where", ["<=", 49, total]],
+    "count-where aggregation with left-hand-side literal",
+  ],
+  [
+    "CountIf([Total] + [Tax] < 52)",
+    ["count-where", ["<", ["+", total, tax], 52]],
+    "count-where aggregation with an arithmetic operation",
+  ],
   // should not compile:
-  ["Sum(Count)", undefined, "aggregation nested inside another aggregation"],
+  // ["Sum(Count)", undefined, "aggregation nested inside another aggregation"],
```

Comment:
Does this not fail anymore or did the message change?

Reply:
Looks like this is a regression as the updated grammar doesn't know about nested aggregation (it should be handled post-parsing). I'll separate the test to a different group to indicate known regression (which shall be fixed in the follow-up PR).